docs/resources/credential.md

```diff
@@ -72,19 +72,23 @@ resource "tanzu-mission-control_credential" "img_reg_cred" {
 
 # Cluster proxy credential
 
+### NOTE:
```

Comment:
Is this added as part of template? I do not see that temple file changes here

Reply:
Missed it, added it now